### PR TITLE
[rps][wallet] Mirror vscode settings for packages migrated to eslint

### PR DESCRIPTION
**Pull request by geoknee**
- allows vscode eslint plugin pickup settings file correctly

### CODE REVIEW

**Comment by snario** [comment]
Is any package not on `eslint`? If not we could use a glob

**Comment by andrewgordstewart** [comment]
> Is any package not on `eslint`? If not we could use a glob

Why is it important that all packages use `eslint`?

**Comment by geoknee** [comment]
`app-wallet-interface` is not, for example. I added these packages only because they had a `.eslintrc.js` file.

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     {"directory": "packages/hub", "changeProcessCWD": true},
     {"directory": "packages/jest-gas-reporter", "changeProcessCWD": true},
     {"directory": "packages/nitro-protocol", "changeProcessCWD": true},
-    {"directory": "packages/channel-client", "changeProcessCWD": true}
+    {"directory": "packages/channel-client", "changeProcessCWD": true},
```
Comment (kerzhner) [inline .vscode/settings.json]: Nit, might be nice to keep the package name list organized alphabetically.

**Pull request merged**